### PR TITLE
[WIP] Migrate particip tables

### PR DESCRIPTION
**Pull request by dartcafe**
Migrate `polls_particip`  and `polls_particip_text` into `polls_options`. --> #255 

- [x] Create table `polls_options`
- [x] Migrate content from `polls_particip`  and `polls_particip_text`
- [x] Delete `polls_particip`  and `polls_particip_text`
- [x] Create table `polls_votes`
- [x] Migrate content from `polls_txts`  and `polls_dts`
- [x] Delete `polls_txts`  and `polls_particip_txt`
- [x] Update Mappers and DB in lib
- [x] Update function calls in templates
- [x] Add/update tests
- [ ] Tests

### CODE REVIEW

**Comment by dartcafe** [comment]
Errors occur. Still WIP

File: lib/Controller/PageController.php [hunk 1]
```diff
@@ -52,19 +48,17 @@
 use OCP\IUserManager;
 use OCP\Mail\IMailer;
 use OCP\Security\ISecureRandom;
-use OCP\User;
+use OCP\User; //depricated
```
Comment (splitt3r) [inline lib/Controller/PageController.php]: We should remove that comment. Perhaps add sth. like `// TODO: Replace with ...`

Comment (dartcafe) [reply lib/Controller/PageController.php]: It just seems so... 😁

Comment (dartcafe) [reply lib/Controller/PageController.php]: Strange, that there was no error.

Comment (dartcafe) [reply lib/Controller/PageController.php]: I updated the code. Hope, everything is fine now.

**Comment by splitt3r** [comment]
Travis doesn look very happe :D activating the app already fails

**Comment by dartcafe** [comment]
Should be happy now. :-)

**Comment by dartcafe** [comment]
So. Squash and merge?

**Comment by splitt3r** [comment]
I will habe another look at it sometime next week. But you can already merge if you want to.

**Pull request merged**